src/plugins/home/public/application/components/use_case_card.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { ContentManagementPluginStart } from '../../../../content_management/public';
+
+export const registerUseCaseCard = (
+  contentManagement: ContentManagementPluginStart,
+  {
+    target,
+    order,
+    id,
+    title,
+    description,
+    icon,
+  }: {
+    target: string;
+    order: number;
+    id: string;
+    title: string;
+    description: string;
+    icon: string;
+  }
+) => {
+  contentManagement.registerContentProvider({
```

Comment:
Just to confirm that it's just a plain card without link?

Reply:
Hi，2 overview page for observability and security-analysis haven't been finished, but I will make it clickable and set the navigation. Thanks